src/timeevolution/timeseries.jl

```diff
@@ -0,0 +1,326 @@
+export evolve!, evolve, construct, extrapolate, timeseries!, timeseries
```

Comment:
maybe `extrapolate` should't be exported? We aren't going to document it anyways, right?